Bottom: 8e9687339d37116aa9006499460eb473ac3e647a
Top:    2ddbc42332c1ad49612cee9f1228cd48f0e16bd7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 07:26:48 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 32036d7..4f24674 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -188,7 +188,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
